commitlint.config.ts

```diff
@@ -57,11 +60,35 @@ module.exports = {
         // * Workflow: detect if wip commit in a branch not named "wip/*" or whose name contains "squashed".
         // * Allow PascalCase word after colon in title (exception to subject-lowercase rule), e.g.: "End2End: TestFixtureSetup refactor"
         // * Detect if commit hash mention in commit msg actually exists in repo.
-        // * Give replacement suggestions in rule that detects too long titles (e.g. and->&, config->cfg, ...)
         // * Detect area(sub-area) in the title that doesn't include area part (e.g., writing (bar) instead of foo(bar))
 
         {
             rules: {
+                'header-max-length-with-suggestions': ({header}: {header:any}) => {
+                    let headerStr = convertAnyToString(header, "header");
+                    let offence = false;
+
+                    let message = `Please do not exceed ${headerMaxLineLength} characters in title.`;
+                    if (headerStr.length > 50) {
```

Comment:
and you're not using headerMaxLineLength here! I guess you forgot

Reply:
Done